lib/src/features/event/logic/map_page_state.dart

```diff
@@ -0,0 +1,49 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
+import 'package:flutter/foundation.dart';
+import 'package:latlong2/latlong.dart';
+
+class MapPageState {
+  LatLng? currentLocation;
+  bool isLoadingCurrentLocation;
+  List<LatLng> events;
```

Comment:
It must be the list of event model, lat and lng are just the params in event model